contents/data/sql/SQL.md

```diff
@@ -0,0 +1,154 @@
+<frontmatter>
+  title: Introduction to SQL
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to SQL
+
+**Author(s): [Amrut Prabhu](https://github.com/amrut-prabhu)**
+
+Reviewers: [Ronak Lakhotia](https://github.com/RonakLakhotia), [Rahul Rajesh](https://github.com/rrtheonlyone)
+
+## What is SQL?
+
+A Relational Database Management System (RDBMS) is a popular database solution used by software applications to manage their data.
+
+**Structured Query Language** (**SQL**) is a programming language that is specifically designed for interacting with an RDBMS. Unlike other languages, SQL doesn't come as a standalone installation.
+Rather, RDBMSs %%like MySQL, Oracle, SQL Server and PostgreSQL%% come with an _implementation_ of SQL. Note that the syntaxes may <tooltip content="in terms of whether the syntax is case-sensitive, the format for specifying date and time, what functions are available out-of-the-box">vary</tooltip> across these RDBMSs.
+
+These RDBMSs come with a GUI application, but can be accessed from the command line as well. Figure 1 shows MySQL Workbench GUI software and MySQL in the command line.
+
+<img src="MySQL_Workbench.png" alt="MySQL Workbench GUI Software" width="57%">
+<img src="MySQL_cli.png" alt="Running MySQL from the command line" width="40%">
```

Comment:
The screenshot is too small; can make bigger. Also, the reader may appreciate the contents of the screenshot even more after they have gone through the query example. Perhaps you can mention (after the SQL query explanation) that there are multiple ways to run queries (GUI, CLI, API,) and show the screenshots there?